libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -251,297 +350,516 @@ class AC_PosControl
+    const Vector3f& get_vel_estimate_NEU_ms() const { return _vel_estimate_neu_ms; }
+
+    // Sets desired velocity in NEU frame in cm/s.
+    // See set_vel_desired_NEU_ms() for full details.
+    void set_vel_desired_NEU_cms(const Vector3f &vel_desired_neu_cms) { set_vel_desired_NEU_ms(vel_desired_neu_cms * 0.01); }
+
+    // Sets desired velocity in NEU frame in m/s.
+    void set_vel_desired_NEU_ms(const Vector3f &vel_desired_neu_ms) { _vel_desired_neu_ms = vel_desired_neu_ms; }
+
+    // Sets desired horizontal (NE) velocity in cm/s.
+    // See set_vel_desired_NE_ms() for full details.
+    void set_vel_desired_NE_cms(const Vector2f &vel_desired_ne_cms) { set_vel_desired_NE_ms(vel_desired_ne_cms * 0.01); }
 
-    /// set_vel_desired_NEU_cms - sets desired velocity in NEU cm/s
-    void set_vel_desired_NEU_cms(const Vector3f &vel_desired_neu_cms) { _vel_desired_neu_cms = vel_desired_neu_cms; }
+    // Sets desired horizontal (NE) velocity in m/s.
+    void set_vel_desired_NE_ms(const Vector2f &vel_desired_ne_ms) { _vel_desired_neu_ms.xy() = vel_desired_ne_ms; }
 
-    /// set_vel_desired_NE_cms - sets the desired horizontal velocity (NE only) in cm/s.
-    void set_vel_desired_NE_cms(const Vector2f &vel_desired_ne_cms) {_vel_desired_neu_cms.xy() = vel_desired_ne_cms; }
+    // Returns desired velocity in NEU frame in cm/s.
+    // See get_vel_desired_NEU_ms() for full details.
+    const Vector3f get_vel_desired_NEU_cms() const { return get_vel_desired_NEU_ms() * 100.0; }
 
-    /// get_vel_desired_NEU_cms - returns desired velocity in cm/s in NEU
-    const Vector3f& get_vel_desired_NEU_cms() const { return _vel_desired_neu_cms; }
+    // Returns desired velocity in NEU frame in m/s.
+    const Vector3f& get_vel_desired_NEU_ms() const { return _vel_desired_neu_ms; }
 
-    // get_vel_target_NEU_cms - returns the target velocity in NEU cm/s
-    const Vector3f& get_vel_target_NEU_cms() const { return _vel_target_neu_cms; }
+    // Returns velocity target in NEU frame in cm/s.
+    // See get_vel_target_NEU_ms() for full details.
+    const Vector3f get_vel_target_NEU_cms() const { return get_vel_target_NEU_ms() * 100.0; }
 
-    /// set_vel_desired_U_cms - sets desired velocity in cm/s in z axis
-    void set_vel_desired_U_cms(float vel_desired_u_cms) {_vel_desired_neu_cms.z = vel_desired_u_cms;}
+    // Returns velocity target in NEU frame in m/s.
+    const Vector3f& get_vel_target_NEU_ms() const { return _vel_target_neu_ms; }
 
-    /// get_vel_target_U_cms - returns target vertical speed in cm/s
-    float get_vel_target_U_cms() const { return _vel_target_neu_cms.z; }
+    // Sets desired vertical velocity (Up) in cm/s.
+    // See set_vel_desired_U_ms() for full details.
+    void set_vel_desired_U_cms(float vel_desired_u_cms) { set_vel_desired_U_ms(vel_desired_u_cms * 0.01); }
+
+    // Sets desired vertical velocity (Up) in m/s.
+    void set_vel_desired_U_ms(float vel_desired_u_ms) {_vel_desired_neu_ms.z = vel_desired_u_ms; }
```

Comment:
nitpick: tiny whitespace issue here, could add a space between "{" and "_vel_desired_neu_ms.z"